res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -1417,17 +1416,15 @@ PioneerDDJSX.tempoResetButton = function(channel, control, value, status, group)
 };
 
 PioneerDDJSX.autoLoopButton = function(channel, control, value, status, group) {
-    if (value) {
-        if (engine.getValue(group, "loop_enabled")) {
-            engine.setValue(group, "reloop_toggle", true);
-        } else {
-            engine.setValue(group, "beatloop_activate", true);
-        }
+    if (engine.getValue(group, "loop_enabled")) {
+        engine.setValue(group, "reloop_toggle", value);
+    } else {
+        engine.setValue(group, "beatloop_activate", value);
```

Comment:
Both code parts are working, but yours is correct.
I also realized another "maybe bug" which has nothing to do with my controller mapping, as it doesn't work on the skin as well. If I load a track and do a beatloop it stays marked in the waveform at the next start of Mixxx. But at this next start I could't do the reloop_toggle. I have to make a new beat loop to be able to use reloop_toggle. I could load other tracks and reload that track with the beat loop again and the reloop_toggle is working. But on the next start of Mixxx it isn't working anymore on this track with this previously created beat loop until I make a new beat loop. Before reloop_toggle I used reloop_exit and it worked in that situation.